$OpenBSD: patch-configure,v 1.1 2012/09/06 11:47:24 ajacoutot Exp $

XXX add configure option and push upstream
Unset HAVE_HW_DECODER_SUPPORT; it depends on gstreamer-plugins-bad and
libva and needs a supported driver backend:
http://www.freedesktop.org/wiki/Software/vaapi

--- configure.orig	Wed Sep  5 16:35:25 2012
+++ configure	Wed Sep  5 16:47:00 2012
@@ -13630,79 +13630,6 @@ fi
 
 
 
-pkg_failed=no
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for HW" >&5
-$as_echo_n "checking for HW... " >&6; }
-
-if test -n "$HW_CFLAGS"; then
-    pkg_cv_HW_CFLAGS="$HW_CFLAGS"
- elif test -n "$PKG_CONFIG"; then
-    if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"gstreamer-basevideo-\$GST_MAJORMINOR >= \$GST_PLUGINS_BAD_REQ_VERSION\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "gstreamer-basevideo-$GST_MAJORMINOR >= $GST_PLUGINS_BAD_REQ_VERSION") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_HW_CFLAGS=`$PKG_CONFIG --cflags "gstreamer-basevideo-$GST_MAJORMINOR >= $GST_PLUGINS_BAD_REQ_VERSION" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
- else
-    pkg_failed=untried
-fi
-if test -n "$HW_LIBS"; then
-    pkg_cv_HW_LIBS="$HW_LIBS"
- elif test -n "$PKG_CONFIG"; then
-    if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"gstreamer-basevideo-\$GST_MAJORMINOR >= \$GST_PLUGINS_BAD_REQ_VERSION\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "gstreamer-basevideo-$GST_MAJORMINOR >= $GST_PLUGINS_BAD_REQ_VERSION") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_HW_LIBS=`$PKG_CONFIG --libs "gstreamer-basevideo-$GST_MAJORMINOR >= $GST_PLUGINS_BAD_REQ_VERSION" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
- else
-    pkg_failed=untried
-fi
-
-
-
-if test $pkg_failed = yes; then
-   	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-
-if $PKG_CONFIG --atleast-pkgconfig-version 0.20; then
-        _pkg_short_errors_supported=yes
-else
-        _pkg_short_errors_supported=no
-fi
-        if test $_pkg_short_errors_supported = yes; then
-	        HW_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "gstreamer-basevideo-$GST_MAJORMINOR >= $GST_PLUGINS_BAD_REQ_VERSION" 2>&1`
-        else
-	        HW_PKG_ERRORS=`$PKG_CONFIG --print-errors "gstreamer-basevideo-$GST_MAJORMINOR >= $GST_PLUGINS_BAD_REQ_VERSION" 2>&1`
-        fi
-	# Put the nasty error message in config.log where it belongs
-	echo "$HW_PKG_ERRORS" >&5
-
-	 have_hw_decoder_support=no
-elif test $pkg_failed = untried; then
-     	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-	 have_hw_decoder_support=no
-else
-	HW_CFLAGS=$pkg_cv_HW_CFLAGS
-	HW_LIBS=$pkg_cv_HW_LIBS
-        { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-
-                   have_hw_decoder_support=yes
-
-$as_echo "#define HAVE_HW_DECODER_SUPPORT 1" >>confdefs.h
-
-
-fi
 
 
 
